cli/dcoscli/task/main.py

```diff
@@ -148,6 +150,17 @@ def _log(follow, completed, lines, task, file_):
                 raise DCOSException(msg)
         raise DCOSException('No matching tasks. Exiting.')
 
+    if file_ in ('stdout', 'stderr') and log.dcos_log_enabled():
+
+        try:
```

Comment:
since we already checked for new the new logging above, we shouldn't need this try/catch.